extensions/amp-story/1.0/amp-story-page-attachment.js

```diff
@@ -241,7 +250,15 @@ export class AmpStoryPageAttachment extends DraggableDrawer {
         <span class="i-amphtml-story-page-attachment-remote-title"><span ref="openStringEl"></span><span ref="urlStringEl"></span></span>
         <svg class="i-amphtml-story-page-attachment-remote-icon" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48"><path d="M38 38H10V10h14V6H10c-2.21 0-4 1.79-4 4v28c0 2.21 1.79 4 4 4h28c2.21 0 4-1.79 4-4V24h-4v14zM28 6v4h7.17L15.51 29.66l2.83 2.83L38 12.83V20h4V6H28z"></path></svg>
       </a>`;
-    const hrefAttr = this.element.getAttribute('href');
+
+    // For backwards compatibility if element is amp-story-page-outlink.
+    const hrefAttr =
+      this.element.tagName === 'AMP-STORY-PAGE-OUTLINK'
+        ? this.element.querySelector('a').getAttribute('href')
+        : this.element.getAttribute('href');
```

Comment:
Suggestion: you have 2 or 3 different places where you duplicate the logic to get the `href`, up to you to see if it's similar enough to factorize with a helper

Reply:
I don't think they're similar enough. This one gets the href where as the other one checks if the href exists.
This could be refactored into a variable `this.href_` that's a `?string`, and it could replace `this.type_`.
That would require non-experiment codepaths to be edited, which doesn't feel right. 
I think this could be cleaned up after launch when we can remove non-experiment code.